conan/cli/formatters/graph/info_graph_html.py

```diff
@@ -312,6 +316,10 @@
                 network.setData(data);
                 network.redraw();
                 network.moveTo({position: viewPos, scale: scale});
+                // If we have found a package to focus, we need to move the view
```

Comment:
what if multiple occurrences of a package? (very common use case)

Reply:
Arbitrarly I chose for the latest matching one to be focused